apps/consumer-client/pages/zuzalu-examples/uuid-proof.tsx

```diff
@@ -16,9 +18,26 @@ import { requestProofFromPassport } from "../../src/util";
  * party developer.
  */
 export default function Page() {
-  const pcdStr = usePassportPCD();
+  const [passportPCDStr, passportPendingPCDStr] = usePassportResponse();
+  const [serverProving, setServerProving] = useState(false);
+
+  const [pcdStr, setPCDStr] = useState("");
   const { signatureProof, signatureProofValid } =
     useSemaphoreSignatureProof(pcdStr);
+  const [pendingPCDStatus, serverPCDStr] = usePendingPCD(
+    passportPendingPCDStr,
+    PASSPORT_SERVER_URL
+  );
+
+  // multiplexer for client-side PCD vs server-side PendingPCD
+  useEffect(() => {
+    console.log(passportPCDStr);
+    if (passportPCDStr) {
+      setPCDStr(passportPCDStr);
+    } else if (serverPCDStr) {
+      setPCDStr(serverPCDStr);
+    }
+  }, [passportPCDStr, passportPendingPCDStr, serverPCDStr]);
```

Comment:
ideally we can factor out this multiplexing into a separate hook (i see you've duplicated it a bunch), but we can do that in a later PR.